app/src/components/modal/modal_airdrop/airdrop_card.tsx

```diff
@@ -91,24 +92,26 @@ const AirdropCard = (props: Props) => {
             </AirdropButton>
           )}
         </TopSection>
-        <BottomSection>
-          <BottomSectionTextWrapper>
-            {displayButtons ? (
-              <BottomSectionTextWrapper>
-                <TYPE.bodyMedium color={'text1'}>Claim OMN token</TYPE.bodyMedium>
-                <TYPE.bodyRegular color={'text2'}>check address for claimable OMN</TYPE.bodyRegular>
-              </BottomSectionTextWrapper>
-            ) : (
-              <BottomSectionTextWrapper>
-                <TYPE.bodyRegular color={'text2'}>
-                  Enter an address to trigger a OMN claim. If the address has any claimable OMN it will be sent to them
-                  on submission.
-                </TYPE.bodyRegular>
-              </BottomSectionTextWrapper>
-            )}
-          </BottomSectionTextWrapper>
-          {displayButtons && <AirdropButton onClick={onCheckAddress}>Check</AirdropButton>}
-        </BottomSection>
+        {displayCheckAddress && (
+          <BottomSection>
+            <BottomSectionTextWrapper>
+              {displayButtons ? (
+                <BottomSectionTextWrapper>
+                  <TYPE.bodyMedium color={'text1'}>Claim OMN token</TYPE.bodyMedium>
+                  <TYPE.bodyRegular color={'text2'}>check address for claimable OMN</TYPE.bodyRegular>
+                </BottomSectionTextWrapper>
+              ) : (
+                <BottomSectionTextWrapper>
+                  <TYPE.bodyRegular color={'text2'}>
+                    Enter an address to trigger a OMN claim. If the address has any claimable OMN it will be sent to
+                    them on submission.
+                  </TYPE.bodyRegular>
+                </BottomSectionTextWrapper>
+              )}
+            </BottomSectionTextWrapper>
+            {displayButtons && <AirdropButton onClick={onCheckAddress}>Check</AirdropButton>}
+          </BottomSection>
+        )}
```

Comment:
I was just looking at his section can you just replace it with this...It should look the same but little bit more efficience and clean.
`  {displayCheckAddress && (
          <BottomSection>
            {displayButtons ? (
              <>
                <BottomSectionTextWrapper>
                  <TYPE.bodyMedium color={'text1'}>Claim OMN token</TYPE.bodyMedium>
                  <TYPE.bodyRegular color={'text2'}>check address for claimable OMN</TYPE.bodyRegular>
                </BottomSectionTextWrapper>
                <AirdropButton onClick={onCheckAddress}>Check</AirdropButton>
              </>
            ) : (
              <TYPE.bodyRegular color={'text2'}>
                Enter an address to trigger a OMN claim. If the address has any claimable OMN it will be sent to them on
                submission.
              </TYPE.bodyRegular>
            )}
          </BottomSection>
        )}`